pkgs/applications/office/ledger/default.nix

```diff
@@ -50,7 +50,7 @@ stdenv.mkDerivation rec {
   '';
 
   meta = with lib; {
-    broken = (stdenv.isLinux && stdenv.isAarch64) || stdenv.isDarwin;
+    broken = (stdenv.isLinux && stdenv.isAarch64);
```

Comment:
```suggestion
    broken = stdenv.isLinux && stdenv.isAarch64;
```

Reply:
Thanks for the catch. I removed the unnecessary parents and re-pushed.